pkg/3scale/amp/operator/apicast_reconciler.go

```diff
@@ -125,6 +135,25 @@ func (r *ApicastReconciler) Reconcile() (reconcile.Result, error) {
 	return reconcile.Result{}, nil
 }
 
+func (r *ApicastReconciler) apicastProductionWorkersEnvVarMutator(desired, existing *appsv1.DeploymentConfig) bool {
```

Comment:
Not sure this is the best way to do this. wdyt @miguelsorianod ?

Reply:
See https://github.com/3scale/3scale-operator/pull/548/files#r553952791 comment

Reply:
Updating for a new implementation based on reconciliation of a single env var

Reply:
:+1: